app/models/hotel.rb

```diff
@@ -1,3 +1,4 @@
 class Hotel < ApplicationRecord
   has_many :rooms
+  has_many :guests, through: :guest_rooms
```

Comment:
This `Hotel` model wouldn't have access to `:guests` through `:guest_rooms` since this model does not have access to the `GuestRoom` model. Line 3 can be removed.

If you did want to get to `:guests` from this model then you would have to go through the `Room` model to do so.

```ruby
has_many :guests, through: :rooms
```